tst/server.ts

```diff
@@ -0,0 +1,30 @@
+import express from 'express';
+import * as bodyParser from 'body-parser';
+import cors from 'cors';
+import { emailRouter } from '../src/routes/email';
+import { authRouter } from '../src/routes/auth';
+import { ensureAuthenticated } from '../src/utils/auth/auth-utils';
+import applicantRouter from '../src/routes/applicant';
+import sheetsRouter from '../src/routes/sheets';
+import gradeRouter from '../src/routes/grade';
+
+export const server = express();
+
+// Enable Cross Origin Resource Sharing
+server.use(cors());
+
+// parse application/json
+server.use(bodyParser.json());
+server.use(bodyParser.urlencoded({ extended: true }));
+
+server.use('/api/auth', authRouter);
+server.use('/api/email', ensureAuthenticated(), emailRouter);
+
+server.use(express.json());
+server.use('/applicant', applicantRouter);
+server.use('/sheets', sheetsRouter);
+server.use('/grade', gradeRouter);
+
+server.get('/', async (req, res) => {
+    res.send('Hello World!');
+});
```

Comment:
What is being done here that isn't done in src/server.ts?

Reply:
Basically src/server.ts listens to one port(4000), but we need to start a server for each test, so this would prompt the "port in use" issue with multiple test files. This would also make us export the server(app) explicitly.